root/edit/edit_header.tt

```diff
@@ -73,8 +78,15 @@
 
     <p class="subheader">
         <span class="prefix">~</span>
-        [% l('Edit by {editor}',
-            { editor => link_entity(edit.editor) }) %]
-        [%- editor_type_info(edit.editor) -%]
+        [%~ IF c.user_exists ~%]
+            [% l('Edit by {editor}',
+                { editor => link_entity(edit.editor) }) %]
+            [%- editor_type_info(edit.editor) -%]
+        [%~ ELSE ~%]
+            [% # We show editor type since knowing if it's, say, a bot edit is useful %]
+            [% l('Edit by a hidden editor') ~%]
```

Comment:
This looked/sounded kinda weird to me, like everyone hid themselves. Something like "Editor hidden (log in to see who)" seems more straightforward, maybe?